src/providers/ConfigurationProvider.ts

```diff
@@ -1,10 +1,13 @@
-import { workspace, WorkspaceConfiguration } from "vscode";
+import { workspace, WorkspaceConfiguration, WorkspaceFolder } from "vscode";
 
 export class ConfigurationProvider {
   public configuration: WorkspaceConfiguration = null;
 
-  constructor() {
-    this.configuration = workspace.getConfiguration("javascript-test-runner");
+  constructor(rootPath: WorkspaceFolder) {
+    this.configuration = workspace.getConfiguration(
+      "javascript-test-runner",
+      rootPath.uri
```

Comment:
rootPath.uri will be alway `undefined` since passed pbject is a string, and not a WorkspaceFolder